docker/release/Dockerfile

```diff
@@ -13,14 +13,30 @@ RUN cat rust-toolchain | xargs rustup toolchain install
 RUN apt-get update && \
     apt-get install -y pkg-config && \
     apt-get install -y protobuf-compiler && \
-    apt-get install -y libssl-dev
+    apt-get install -y libssl-dev && \
+    apt-get install -y libclang-dev
 
 COPY Cargo.toml Cargo.lock ./
 COPY ./core/Cargo.toml ./core/Cargo.toml
-COPY ./circuits ./circuits
-COPY ./crypto ./crypto
+
+COPY ./arbitrum-client ./arbitrum-client
 COPY ./circuit-macros ./circuit-macros
-COPY ./integration-helpers ./integration-helpers
+COPY ./circuit-types ./circuit-types
+COPY ./circuits ./circuits
+COPY ./common ./common
+COPY ./config ./config
+COPY ./constants ./constants
+COPY ./external-api ./external-api
+COPY ./gossip-api ./gossip-api
+COPY ./renegade-crypto ./renegade-crypto
+COPY ./srs ./srs
+COPY ./state ./state
+COPY ./statev2 ./statev2
+COPY ./system-bus ./system-bus
+COPY ./task-driver ./task-driver
+COPY ./test-helpers ./test-helpers
+COPY ./util ./util
+COPY ./workers ./workers
```

Comment:
we can just copy in the whole repo now, no need to do one by one. this was done before these were individual crates

Reply:
ack